src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 1) {
+    return 0;
+  }
+
+  if (days < 3) {
+    return days * 40;
+  }
+
+  if (days < 7 && days >= 3) {
+    return days * 40 - 20;
+  } else {
```

Comment:
3. [CODE STYLE] - You should return the value immediately in `if` statements. And don't use `else` after return statement. If `return` is performed - code after it won't be executed anyway.